### PR TITLE
make proxy configuration as an option

### PR DESCRIPTION
**Pull request by jkhelil**

### CODE REVIEW

File: config/crd/bases/jenkins.io_jenkins.yaml [hunk 1]
```diff
@@ -5272,6 +5276,10 @@ spec:
                     volumeSize:
                       type: string
                   type: object
+                proxyConfigurationEnabled:
```
Comment (akram) [inline config/crd/bases/jenkins.io_jenkins.yaml:5281]: it needs better explanation: It seems that you are taking the http proxy configuration from global config; if it is the case, it should appear in the field documentation and in the documentation.

Comment (jkhelil) [reply config/crd/bases/jenkins.io_jenkins.yaml:5281]: done

**Pull request merged**